From: Gordon Ball <gordon@chronitis.net>
Date: Sat, 6 Nov 2021 14:15:57 +0000
Subject: Fix test_serverextensions for jupyter-core 4.9

The set-up for these tests make assertions about jupyter path, the logic
for which changed in jupyter-core 4.9, breaking the assertions (in a way
which probably doesn't actually break the subsequent tests).
---
 notebook/tests/test_serverextensions.py | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/notebook/tests/test_serverextensions.py b/notebook/tests/test_serverextensions.py
index 8986479..5e0795a 100644
--- a/notebook/tests/test_serverextensions.py
+++ b/notebook/tests/test_serverextensions.py
@@ -94,9 +94,10 @@ class MockEnvTestCase(TestCase):
             p.start()
             self.addCleanup(p.stop)
         # verify our patches
-        self.assertEqual(paths.jupyter_config_path(), [self.config_dir] + self.system_config_path)
-        self.assertEqual(extensions._get_config_dir(user=False), self.system_config_dir)
-        self.assertEqual(paths.jupyter_path(), [self.data_dir] + self.system_path)
+        # the exact logic tested here changed in jupyter-core 4.9, adding extra paths
+        # self.assertEqual(paths.jupyter_config_path(), [self.config_dir] + self.system_config_path)
+        # self.assertEqual(extensions._get_config_dir(user=False), self.system_config_dir)
+        # self.assertEqual(paths.jupyter_path(), [self.data_dir] + self.system_path)
     
     def tearDown(self):
         for modulename in self._mock_extensions:
